pytorch/inference/docker/1.7.0/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,107 @@
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
+
+ARG PYTHON_VERSION=3.6.10
+ARG TS_VERSION="0.2.1=py36_0"
+ARG PT_INFERENCE_URL=https://aws-pytorch-binaries.s3-us-west-2.amazonaws.com/r1.7.0_inference/20201023-100313/30525ef129b6def810885e5f3b2be27e3e1dd813/cpu/torch-1.7.0-cp36-cp36m-manylinux1_x86_64.whl
```

Comment:
Temporary change to test `libmpi` issue: `https://choidong-dev.s3-us-west-2.amazonaws.com/1.7.0/cpu/inference/torch-1.7.0a0-cp36-cp36m-manylinux1_x86_64.whl`